src/main/resources/liquibase/tenant/scripts/2020-08-20--12-00-enhance-set-table.xml

```diff
@@ -0,0 +1,40 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+
+  <changeSet id="2020-08-20-drop-filtering-conditions-column"
+             author="Illia Daliek">
+    <dropTable cascadeConstraints="true"
+               schemaName="${database.defaultSchemaName}"
+               tableName="filtering_condition"/>
+  </changeSet>
+
+  <changeSet id="2020-08-20-add-filtering-conditions-column-into-set-table"
+             author="Illia Daliek">
+    <addColumn schemaName="${database.defaultSchemaName}"
+               tableName="set">
+      <column name="filtering_conditions"
+              type="text"/>
```

Comment:
Here the text data type is used instead of  json cause the jooq-vertx-classic-reactive library has an issue with generating sql query. When json or jsonb data types are used there is a necessety of using data types binding and convertes and here the problems take place. Actually at last library release (5.1.2) it says that jooq can process JSON and JSONB types directly but in real it dosn't. So, there is no way for processing such data types wihout using special bindings, but   there is a problem with redundant brace. Actually i'm gonna report this issue to jooq developers and hope we will be able to use json and jsonb columns whithout any chalanges soon.